Extractor/UAClient.cs

```diff
@@ -234,18 +231,9 @@ private async Task StartSession(int timeout = -1)
 
             // A restarted Session might mean a restarted server, so all server-relevant data must be cleared.
             // This includes any stored NodeId, which may refer to an outdated namespaceIndex
```

Comment:
This comment does not make too much sense with the LoadAppConfig call

Reply:
No... I'll remove it.